golang/cosmos/x/vbank/vbank_test.go

```diff
@@ -85,12 +86,12 @@ func validateBalanceUpdate(vbu VbankBalanceUpdate) error {
 
 // decodeBalances unmarshals a JSON-encoded vbankBalanceUpdate into normalized balances.
 // A nil input returns a nil balances.
-func decodeBalances(encoded []byte) (balances, uint64, error) {
-	if encoded == nil {
+func decodeBalances(encoded string) (balances, uint64, error) {
+	if encoded == "" {
```

Comment:
There's a subtle difference here, because `[]byte("") == nil` is false. Assuming that change is acceptable, the doc comment should be updated accordingly.